librz/core/meson.build

```diff
@@ -11,9 +11,13 @@ rz_core_sources = [
   'cautocmpl.c',
   'cbin.c',
   'cconfig.c',
+  'ccore.c',
```

Comment:
`ccore.c` seems a bit recursive :D `c` stands for `core`, so that would be `corecore`! I'd move whatever you placed in `ccore.c` into `core.c` for now.